frontend/src/views/Header.vue

```diff
@@ -1,11 +1,29 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+const router = useRouter();
+
+function goToAbout() {
+  router.push({ name: 'About' })
+}
+
+function goToContact() {
+  router.push({ name: 'Contact' })
+}
+
+function goToData() {
+  router.push({ name: 'Data' })
+}
+
+</script>
+
 <template>
   <div id="header">
     <span id="logo">Logo</span>
-    <span class="header-links" id="about">About</span>
+    <span class="header-links" id="about" @click="goToAbout">About</span>
```

Comment:
This works, but I wonder if it would be cleaner to use [`<router-link>`](https://router.vuejs.org/guide/#router-link). It may change the formatting though since it implicitly adds an `a` tag. But a little CSS can fix that up.

What are your thoughts? We could also take care of this in a follow-up PR.

Reply:
Perfect, just what I was looking for. I'm struggling to override the implicit `a` tag, do you have suggestions? 

In `Header.vue` I can add to the bottom e.g.

```
a {
  color: $whatever
}
```

But I don't want the formatting to apply to both the `header-links` and the `button`. Not sure how to adjust the syntax.

Reply:
You can see an example in `/src/home/Stories.vue`.